templates/index.html

```diff
@@ -4,8 +4,12 @@
         <title>short.rs</title>
         <link href="https://fonts.googleapis.com/css?family=Big+Shoulders+Display&display=swap" rel="stylesheet">
         <style>
+        * {
+          box-sizing: border-box;
+        }
+        
             body {
-              background-color: #B8C982;
+              background-color: #7FA5BF;
             }
             .container {
```

Comment:
I know it's not your code, but it would be great if you could add newlines between each block of css code :)